tests/components/seventeentrack/__init__.py

```diff
@@ -1 +1,26 @@
 """Tests for the seventeentrack component."""
+
+from datetime import timedelta
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.seventeentrack.sensor import DEFAULT_SCAN_INTERVAL
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+from homeassistant.setup import async_setup_component
+
+from tests.common import async_fire_time_changed
+
+
+async def init_integration(hass: HomeAssistant, config: ConfigType):
+    """Set up the seventeentrack integration in Home Assistant."""
+    assert await async_setup_component(hass, "sensor", config)
+    await hass.async_block_till_done()
+
+
+async def _goto_future(hass: HomeAssistant, freezer: FrozenDateTimeFactory):
+    """Move to future."""
+    for _ in range(2):
```

Comment:
so why do we do this twice? (or 3 times if I see correctly)?

Reply:
I removed the loop. But some tests need to call this twice as it seems the sensor first becomes "unavailable" before gets removed

Reply:
Would it maybe make sense to skip time, check that the state is unavailable and then skip again and check if it's removed?

Reply:
Yes I've added that in the test

Reply:
@joostlek did you mean to check here or is it enough in the test I've added to it?